packages/marble-datatable/src/Datatable.js

```diff
@@ -345,13 +345,23 @@ class Datatable extends Component {
     return core.isObject(data) && 'columns' in data && 'type' in data;
   }
 
+  /**
+   * Clone JSON data
+   * @param {*} data
+   * @return {*}
+   */
+  cloneJSONData(data) {
+      return JSON.parse(JSON.stringify(data));
+  }
+
   /**
    * Setter for the `data` state property.
    * @param {!Object}
```

Comment:
Could you add the parameter name here, please? I think we forgot this when it was created.

```javascript
/*
 * @param {!Object} givenData <<
```